sh/travis.sh

```diff
@@ -1,14 +1,40 @@
 ##
-## This script is for Travis CI to build the default user to for the database.
+## This script is for Travis CI to configure and install required softwares.
 ##
+
+## mysql is installed by default on travis, so we can set mysql server charset
+echo -e "
+[client]
+default-character-set=utf8
+
+[mysql]
+default-character-set=utf8
+
+[mysqld]
+collation-server=utf8_unicode_ci
+init-connect='SET NAMES utf8'
+character-set-server=utf8
+sql-mode=\"STRICT_ALL_TABLES\" " | sudo tee -a /etc/mysql/conf.d/99_custom_travis.cnf
+
+## set database environment
 set -euo pipefail
 
 set -a
 source .env.development
 set +a
 
-mysql -h $DB_HOST -u root -e "SET GLOBAL sql_mode='STRICT_ALL_TABLES';"
-
 # setup usernames and permissions
 mysql -h $DB_HOST -u root -e "GRANT ALL PRIVILEGES ON *.* TO '$DB_USER'@'$DB_HOST' IDENTIFIED BY '$DB_PASS' WITH GRANT OPTION;"
 mysql -h $DB_HOST -u root -e "FLUSH PRIVILEGES;"
+
+## install and configure softwares
+export CHROME_BIN=/usr/bin/google-chrome
+/sbin/start-stop-daemon --start --quiet --pidfile /tmp/custom_xvfb_99.pid --make-pidfile --background --exec /usr/bin/Xvfb -- :99 -ac -screen 0 1280x1024x16
+sudo apt-get update -qq
+wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
```

Comment:
Let's try removing Google Chrome from out build completely.  It looks like [Travis supports google chrome as an addon](https://docs.travis-ci.com/user/chrome).  I propose you remove all references to Chrome here and then add:
```yml
addons:
  chrome: stable
```
to the `.travis.yml` file.